### DISCUSSION TITLE
unknown function: os.abs_path

### DISCUSSION THREAD

**Discussion by JoyceBabu** [vlang/v#15605] [General]
I am trying to compile [`finder_utils.v`](https://raw.githubusercontent.com/vlang/v/master/cmd/tools/vwhere/finder_utils.v).

The compilation is failing with error

> unknown function: os.abs_path

What am I doing wrong?

**Comment by JalonSolov** [comment]
What version of V do you have?  I just tried with latest version, and did not see a problem.

**Reply by JoyceBabu** [reply]
Running `v up` fixed the issue. Thanks.
